scripts/README.md

```
@@ -0,0 +1,623 @@
+
+**Usage:**
+
+```bash
+./scripts/run-rpc.sh
+```
+
+**Environment Variables:**
+
+- `BUILD_CONFIGURATION` - Build configuration (default: Debug)
+- `ASPNETCORE_ENVIRONMENT` - ASP.NET Core environment (default: Development)
+- `ASPNETCORE_URLS` - Listen URLs (default: http://localhost:8081)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)